src/suricata-common.h

```diff
@@ -493,6 +493,10 @@ typedef enum {
     LOGGER_JSON_STREAM,
     LOGGER_JSON_ARP,
 
+    /* An ID that can be used by loggers registered by plugins and/or
+     * library users. */
+    LOGGER_USER,
```

Comment:
Would multiple loggers using the same ID be bad for transaction loggers, I think they are the only loggers using the logger_id for some state management. @catenacyber Have you looked at making these dynamic in any of your work?

Reply:
I did not touch these IDs...